Source/Csla/Serialization/Mobile/CustomSerializers/PocoSerializer.cs

```diff
@@ -0,0 +1,40 @@
+﻿//-----------------------------------------------------------------------
+// <copyright file="ClaimsPrincipalFormatter.cs" company="Marimer LLC">
+//     Copyright (c) Marimer LLC. All rights reserved.
+//     Website: https://cslanet.com
+// </copyright>
+// <summary>A formatter that can serialize and deserialize a POCO</summary>
+//-----------------------------------------------------------------------
+
+using System.Text.Json;
+
+namespace Csla.Serialization.Mobile.CustomSerializers;
+
+/// <summary>
+/// A formatter that can serialize and deserialize a POCO
+/// by using System.Text.Json.
+/// </summary>
+public class PocoSerializer<T> : IMobileSerializer
```

Comment:
As far as I know STJ isn't that good with circular references. But that knowledge is a bit outdated so get that fixed?

Reply:
It is not. If people want full fidelity serialization they'll need to use MobileFormatter, Newtonsoft, or (maybe) protobuf.

This will be a limitation of the default POCO serialization support, as I sure don't want to write and maintain what amounts to a whole _other_ MobileFormatter that can serialize POCO graphs with full fidelity.